### ISSUE TITLE
Mac OS binaries (and others maybe?) are not built

### ISSUE THREAD

**Issue by FloWi** [svenstaro/dummyhttp#390]
Hi!
A colleague recommended your wonderful little helper and I wanted to try it out. First of all thanks a lot for making such a handy tool!

Unfortunately the macOS binaries are not included in the release. 

I saw several failed jobs in the github actions, but I wasn't able to figure out what's wrong, especially since some build seem to work fine. 
I'm a rust-n00b, but luckily `cargo install dummyhttp` ran without problems. Had to reinstall my outdated rust setup though to use arm64 instead of amd64. 

I'll ask around at work next week - some of my colleagues have experience with rust.

**Comment by svenstaro** [comment]
To specify this, I believe you mean that AArch64 OSX binaries aren't built. The AMD64 ones _are_ being uploaded into the release. I'll see whether it's easy to also build the AArch64 ones.

**Comment by FloWi** [comment]
Oh, you're right. Sorry for the confusion. I hope this build target is easy to add.

**Comment by svenstaro** [comment]
Here you go.

**Comment by FloWi** [comment]
Awesome! Thanks a lot, @svenstaro, I really appreciate it. 

Didn't work at first when I downloaded it with Chrome directly from github. When opening it always opened this finder-window saying that this app comes from an untrusted source (or something like that). When I downloaded it with `wget`, everything works fine. Don't know how the sandboxing works on macOS when a browser downloaded something.

<img width="297" alt="image" src="https://user-images.githubusercontent.com/39078/235090022-3f3111ad-9d0b-4553-acc3-9250ceb4536d.png">
